### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.description)

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#12]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI))
#### Target application
[http://localhost:8080/public/do/eo/eomeetingrequest?id=QEApUYrXJLU-3rYDonTP1AdqyXz4-J-ptR_ynA9hjdxOkAZSuDI](http://localhost:8080/public/do/eo/eomeetingrequest?id=QEApUYrXJLU-3rYDonTP1AdqyXz4-J-ptR_ynA9hjdxOkAZSuDI)
#### Element path
.description
#### Snippet
`<div class="description">In order to better facilitate your meeting request, please identify attendees who will be in attendance.</div>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.04 (foreground color: #17a2b8, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 81.0.4044.69
This accessibility issue was found using Accessibility Insights for Web 2.15.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
